security/pkg/server/ca/authenticate/xfcc_authenticator.go

```diff
@@ -98,7 +99,12 @@ func isTrustedAddress(addr string, trustedCidrs []string) bool {
 		}
 	}
 	// Always trust local host addresses.
-	return net.ParseIP(addr).IsLoopback()
+	ip, _, err := net.SplitHostPort(addr)
```

Comment:
why donot split in front, so isInRange can use it

Reply:
done